rumps/rumps.py

```diff
@@ -520,7 +520,7 @@ def __new__(cls, *args, **kwargs):
             return args[0]
         return super(MenuItem, cls).__new__(cls, *args, **kwargs)
 
-    def __init__(self, title, callback=None, key=None, icon=None, dimensions=None, template=None):
+    def __init__(self, title, name=None, callback=None, key=None, icon=None, dimensions=None, template=None):
```

Comment:
Let's remove usages of name, since they are not a requirement for NSSlider.

Repetitive marks of usages incoming.